labs/iot/gps-lab/steps/add-gps-to-pi.md

```diff
@@ -0,0 +1,19 @@
+# Add the GPS sensor to Raspberry Pi
+
+In this step you will set up the GPS sensor to Pi.
+
+## Steps
+
+1. Put the vcc to 2, ground to 6 tx and rx to 8 and 10
```

Comment:
Can we have more details here. Does the GPS come with wires already fitted? Will they have to solder them on? If so, any tips on which color wires to use?